doc/release-notes/5.11-release-notes.md

```diff
@@ -0,0 +1,187 @@
+- For Shibboleth users, the `emailconfirmed` timestamp is now set on login for Shibboleth users and the UI should show "Verified". (Issue #5663, PR #8579)
+- Information about the license selection (or custom terms) is now available in the confirmation popup when contributors click "Submit for Review". Previously, this was only available in the confirmation popup for the "Publish" button, which contributors do not see. (Issue #8561, PR #8691)
+- For installations configured to support multiple languages, controlled vocabulary fields that do not allow multiple entries (e.g. journalArticleType) are now indexed properly. (Issue #8595, PR #8601, PR #8624)
+- Harvesting now works when the Dublin core "language" field is set is set #8139. (Issue #8139, PR #8689)
+- The API endpoint for listing notifications has been enhanced to show the subject, text, and timestamp of notifications. (Issue #8487, PR #8530)
+- The API Guide has been updated to explain that the `Content-type` header is now (as of Dataverse 5.6) necessary to create datasets via native API. (Issue #8663, PR #8676)
+- Admin API endpoints have been added to find and delete dataset templates. (Issue 8600, PR #8706)
+- The BagIt file handler detects and transforms zip files with a BagIt package format into Dataverse data files, validating checksums along the way. See the [BagIt File Handler](https://guides.dataverse.org/en/5.11/installation//config.html#bagit-file-handler) section of the Installation Guide for details. (Issue #8608, PR #8677)
+- For BagIt Export, the number of threads used when zipping data files into an archival bag is now configurable using the `:BagGeneratorThreads` database setting. (Issue #8602, PR #8606)
+- PostgreSQL 14 can now be used (though we've tested mostly with 13). PostgreSQL 10+ is required. (Issue #8295, PR #8296)
+- As always, widgets can be embedded in the `<iframe>` HTML tag, but the HTTP header "Content-Security-Policy" is now being sent on non-widget pages to prevent them from being embedded. (PR #8662)
+- URIs in the the experimental Semantic API has changed (details below). (Issue #8533, PR #8592)
+- Installations running Make Data Count can upgrade to Counter Processor-0.1.04. (Issue #8380, PR #8391)
+- PrimeFaces, the UI framework we use, has been upgraded from 10 to 11. (Issue #8456, PR #8652)
+
+## Notes for Dataverse Installation Administrators
+
+### Identifying Datasets Requiring Terms of Access or Request Access Changes
+
+In support of the change to require either Terms of Access or Request Access for all Restricted Files, we have provided a query to identify datasets in your installation where at least one file has neither Terms of Access or Request Access enabled:
+
+https://github.com/IQSS/dataverse/blob/v5.11/scripts/issues/8191/datasets_without_toa_or_request_access
+
+This will allow you to reach out to those dataset owners as appropriate.
```

Comment:
What happens if people don't do this? Seems like a bigger warning, i.e. users with this problem won't be able to publish until they fix this (just reading the PR) might be useful. (If I understand there's a warning in the UI - is that true for the API as well?)

Reply:
Right. Nothing happens to published versions, but owners are prevented from creating, editing or publishing draft versions until they update the terms/allow access requests.

Reply:
I had a nice chat with @sekmiller and played with the feature a bit and made the warning banner appear. In c302cd626a I tried to improve the release notes.

@qqmyers @sekmiller et al., please take a look. This is the last thing before we move this PR to QA and cut 5.11! 🎉